fi_input:
  topics: 
    - 'shared_control_publisher/foot_input'
    - 'gripperFeedbackToPlatform'
    #- 'surgical_task/foot_input'
    - 'foot_haptic_efforts'
